README.md

```diff
@@ -3,14 +3,14 @@
 An easy, convenient way to automatically update Homebrew.
 
 This script will run `brew update` in the background once every 24 hours (by
-default) until explicitly told to stop, utilising `launchd`.
+default) until explicitly told to stop, utilizing `launchd`.
```

Comment:
Tap uses British English rather than American, so technically not a spelling issue here 😇